module/VuFind/src/VuFind/View/Helper/Root/RecordLinker.php

```diff
@@ -48,6 +48,8 @@
  */
 class RecordLinker extends \Laminas\View\Helper\AbstractHelper
 {
+    use \VuFind\Log\VarDumperTrait;
```

Comment:
Is there a reason you're adding the trait here but not changing any other code?

Reply:
It's a mistake, fixed.